backend/src/main/scala/cromwell/backend/standard/package.scala

```diff
@@ -0,0 +1,18 @@
+package cromwell.backend
+
+import shapeless.{:+:, CNil}
+import wom.callable.AdHocValue
+
+package object standard {
+  object StandardAdHocValue {
+    object AsAdHocValue {
+      def unapply(arg: StandardAdHocValue): Option[AdHocValue] = arg.select[AdHocValue]
+    }
+    object AsLocalizedAdHocValue {
+      def unapply(arg: StandardAdHocValue): Option[LocalizedAdHocValue] = arg.select[LocalizedAdHocValue]
+    }
+  }
```

Comment:
pity these unapplys need to be written out, seems like a common thing one would want to do with coproducts

Reply:
The upside of this is it can be re-used for different purposes, whereas we need a different Poly1 for each sort of "map" we want to do